spec/models/resource_spec.rb

```diff
@@ -52,7 +52,7 @@
     end
 
     context 'when resources are maxed_out or dirty' do
-      before { @resource.update_attributes(dirty: true, attempts: Setting['advanced.attempts'].to_i + 1, try_again_at: 1.minute.ago) }
+      before { @resource.update_attributes(dirty: true, attempts: ENV['attempts'].to_i + 1, try_again_at: 1.minute.ago) }
```

Comment:
Line is too long. [121/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.